meteorite_fall.js

```diff
@@ -19,14 +20,18 @@ function main() {
 // main game functionality
 function loopGame() {
     let randomMeteoriteIndex = Math.floor(Math.random() * meteorites.length);
-
+    isGameOver = false;
```

Comment:
In a comment below I mention removing isGameOver. But in this current code, I don't see the purpose of this line.

Reply:
So line 23 is for restarting the game after it has ended.If you try it out, you'll see that once the bool is set to true, the game won't start over on it's own.

Reply:
I'm confused. So your hypothesis is that the bool isGameOver is necessary if we want to implement the restart feature right? It appears right now the isGameOver bool is never used. It is just set to different values. I guess this makes sense if there is no way to check if a function is being used with an interval or not.

So either we keep the isGameOver bool and use it to redo the setInterval(loopGame, 50). Or maybe we should never really 'break' from loopGame and just add a 
`
if (isGameOver)
    return;
`

I kinda like the second solution better. If you do too go ahead and make this change before merging. All other comments can be resolved.